packages/patternfly-3/patternfly-react/less/dual-list-selector.less

```diff
@@ -117,12 +117,22 @@
     color: white;
   }
 
+  &.disabled {
+    cursor: not-allowed;
+    background: @color-pf-black-150;
+    color: @color-pf-black-500;
```

Comment:
nit. supposed to be in alphabetical order.

Reply:
we haven't been keeping up with that in this repo